mkosi/__init__.py

```diff
@@ -1836,6 +1836,7 @@ def prepare_tree(args: MkosiArgs, root: Path, do_run_build_script: bool, cached:
             root.joinpath("etc/kernel").mkdir(mode=0o755)
 
             root.joinpath("etc/kernel/cmdline").write_text(" ".join(args.kernel_command_line) + "\n")
+            root.joinpath("etc/kernel/entry-token").write_text(f"{args.machine_id}\n")
```

Comment:
I think I understand your reasoning for using the machine id here, but wouldn't the image id not also be a sensible choice, so that in the end this would be sensible to be configurable?

Reply:
Yes eventually it should be configurable. I just figured we could wait with adding that until this is supported somewhat widely